doc/neps/nep-0018-array-function-protocol.rst

```diff
@@ -145,7 +149,9 @@ This will require two changes within the Numpy codebase:
 1. A function to inspect available inputs, look for the
    ``__array_function__`` attribute on those inputs, and call those
    methods appropriately until one succeeds.  This needs to be fast in the
-   common all-NumPy case.
+   common all-NumPy case, and have acceptable performance (no worse than
```

Comment:
I'd argue that "It should also be possible to skip the test altogether".

Reply:
I'm not sure I agree here. This isn't the case for `__array_ufunc__`.

Reply:
It would be if adding `subok=False` didn't cause a bigger delay than checking for `__array_ufunc__`. But perhaps this also suggests that the current phrasing of "it should be fast" is good enough.